content/en/getting-started.md

```diff
@@ -5,6 +5,11 @@ top_graphic: 3
 date: 2020-02-11
```

Comment:
Should this `date` line get updated for translations?

Reply:
Rather than update `date`, I've added a `lastmod` field to match the other documents.